### ISSUE TITLE
AirPlay 2 Compatibility?

### ISSUE THREAD

**Issue by ezadoo** [Ottercast/OtterCastAudioV2#13]
As Shairport-Sync seems to be in the final stages of implementing the AirPlay 2 functionalities, it would be very interesting to know if there is 
a) a way of installing the beta-version of shairport-sync 
and b) how it will perform on OttercastAudioV2, if it is released.

As I currently don’t have an Device myself, as I hoped I would be able to simply buy a completely working OttercastAudio-board.
And if If i do the additional soldering myself, I would like to know, if I could use it later for AirPlay 2 too.

**Comment by Manawyrm** [comment]
> AirPlay 2 functionalities
Yeah, AirPlay 2 is much more involved than AirPlay 1, as it needs strict timing accuracy, would normally also require additional hardware features (but I think we can manage without them).

Generally: The firmware for the Ottercast is built via buildroot from this repository:
https://github.com/Ottercast/buildroot-ottercast-audio
We're already using a custom version of shairport-sync with some mods, which is currently hosted on my GitHub account:
https://github.com/Ottercast/buildroot-ottercast-audio/blob/master/buildroot/package/shairport_sync_pulseaudio/shairport_sync_pulseaudio.mk
This would need to be changed to use the development branch (or the new version when it's released). 

Getting the dev version / v2 to work with PulseAudio in general might be a bit of a pain. Maybe it's time to evaluate Pipewire first (or to switch to plain ALSA outright)... The OttercastAmp (not the AudioV2) also seems to still be suffering from some timing / clock-distribution problem which would probably need to be addressed before this starts working properly. 

Performance-wise: I don't really expect any trouble there. 

... to be direct: Each single one of us in the team have started a new full-time role as an engineer at different companies since we started this project, so time is running a bit low. The OttercastAmp is my daily driver amp and it has reached that status of "good enough, plays music, can worry about other things", which is a dangerous place to be in :smile:
